src/COFF/COFFHandle.jl

```diff
@@ -49,8 +49,8 @@ function readmeta(io::IO, ::Type{H}) where {H <: COFFHandle}
     magic = [read(io, UInt8) for idx in 1:4]
     if any(magic .!= PE_magic)
         msg = """
-        Magic Number 0x$(join(string.(magic, base=16),"")) does not match expected PE
-        magic number 0x$(join("", string.(PE_magic, base=16)))
```

Comment:
How....did this ever work 😕 I guess it never did.  Also in v0.6, when this line
https://github.com/JuliaIO/ObjectFile.jl/blob/058bdba211dc1a04d426006364527cba7aa73251/src/COFF/COFFHandle.jl#L53 [was written](https://github.com/JuliaIO/ObjectFile.jl/commit/03c37b57edd1a06a9c0578e9de8c8fc8eb751d89), it gave the wrong result:
```julia-repl
julia> join("", hex.(UInt8['P','E','\0','\0']))
""
```